sidecar_acceptor.go

```diff
@@ -212,7 +212,7 @@ func (a *SidecarAcceptor) RegisterSidecar(ctx context.Context,
 		ctx, int32(keychain.KeyFamilyMultiSig),
 	)
 	if err != nil {
-		return fmt.Errorf("error deriving multisig key: %v", err)
+		return nil, fmt.Errorf("error deriving multisig key: %v", err)
 	}
 
 	ticket.State = sidecar.StateRegistered
```

Comment:
should we actually make a copy rather then mutating this?

Reply:
Yeah I think I like that path, I made a change elsewhere to start to copy and modify instead of mutating. We've had some bugs on the server-side related to mutation w/ pointer and IMO copying makes the control flow more clear generally.